bundles/binding/org.openhab.binding.denon/src/main/java/org/openhab/binding/denon/internal/DenonBinding.java

```diff
@@ -137,73 +137,75 @@ protected void removeBindingProvider(DenonBindingProvider bindingProvider) {
      */
     @Override
     public void updated(Dictionary<String, ?> config) throws ConfigurationException {
-        if (config != null) {
-            logger.debug("Denon binding updated");
+        logger.debug("Denon binding updated");
 
-            Enumeration<String> keys = config.keys();
+        if (config == null) {
+            logger.warn("Denon binding has no configuration defined. Check the configuration file.");
```

Comment:
`updated` can be called with a `null` `Dictionary`, but this doesn't mean that there is no configuration file.  [Here](https://nakvic-dev.blogspot.ie/2010/11/osgi-declarative-managed-services-and.html) is an example outside the world of openHAB that shows a case where `updated` silently does nothing if passed a `null` `Dictionary`.  There are also many bindings that handle this situation the same silent way.  So I worry that this log message will heighten user anxiety without pointing to an actual problem.